src/elements/content-preview/ContentPreview.js

```diff
@@ -47,6 +47,7 @@ import {
     ERROR_CODE_UNKNOWN,
 } from '../../constants';
 import type { ErrorType } from '../common/flowTypes';
+import ContentSidebar from '../content-sidebar';
```

Comment:
I think this may actually cause issues with how we're doing code splitting. Can we use a more generic type, instead?

Reply:
Ah thanks for pointing this out! Will do 👍